### DISCUSSION TITLE
CABIN DAO - a quick summary

### DISCUSSION THREAD

**Discussion by martinprogrammer** [life-itself/community#983] [Research]
A quick summary of what CABIN is all about.

In a nutshell:

1. Decentralized cities - online & in real life co-living communities with common culture, governance and economy with a high density of creative people
2. A thriving creator economy - creator co-ops and a lab for experimenting with revenue generation
4. Physical locations for creative communities - purchase and development of real estate 

There doesn't appear to be an emphasis on "global change" or "paradigm shift", but strong focus on the project sustainability and financial side of things.

Mission

**Jon Hillis**, the founder of CABIN, shared his vision for the ultimate realisation of the project:

**A world where people can live in flexible ways,** 
**travel to different places,**
**have home in those places,**
**co-live with people who have aligned vision,** 
**maintain this flexibility and** 
**be part of multiple overlapping networks**

---
Cabin is **a place to get together "in real life" with your internet friend**s.

The metaverse is the best place to meet people and find your tribe. But reality is **much higher bandwidth**. Once you’ve made connections online, we believe in coming together in person to deepen relationships and experience the liminal space and time that leads to creative breakthroughs and the best conversations.

---
Social connections

We are building a network city: somewhere to feel at home anywhere in the world. This means people nearby who you want to spend time with. Friends to have over for a dinner party. Spare bedrooms to stay in across the world. People to connect with online and IRL.

these points reinforce the need for online engagement to be the easiest to access value prop, starting with online interactions along a potential path of experiences, some at the far opposite end resulting in people living year round together.

[![Path of engagement](https://global.discourse-cdn.com/standard14/uploads/cabin/optimized/1X/96b136bfb0bd8c39350beb28865bf34be42efda7_2_690x200.png)

---
Collaboration

Creating the conditions for serendipitous in-person collaboration across a diverse internet community (foster unplanned, fortunate encounters and collaboration among people within the community)

---
Generating income

The main way the good citizens of Cabin can make a living online is through contributing their skills and time to its **guilds.**

Borrowing terminology from gaming, guilds serve as a talent network for Cabin-initiated projects. These scoped, one-off projects are called **bounties**. The DAO puts up ₡ABIN token as rewards for completing bounties and the people who complete them are — you guessed it — **bounty hunters**. 

Each guild has its own process for getting bounties approved. Some bounties are for community members only, while others are open to the greater public to “bounty their way” into DAO membership. Cabin is a bounty-first DAO, meaning “freelance” bounty hunters earn just as much or even more than “salaried” core contributors.

---
Knowledge sharing & transparency

To help the people with the best ideas and the ability to execute those ideas rise to the roles in which they’re able to execute, everyone involved has to have a common understanding of the mission and the intent of the mission.

All of Cabin’s core documentation is accessible to anyone; information about their token and governance is on-chain, and all the code they’ve written is open-sourced and reposed online.

---
CABIN census

Members directory and reputation system - physical chip embedded passport cards where members can build identity and reputation

---
CABIN citizenship

Subscription membership that grants access to the City directory and comes with rights and responsibilities

---
Proposals, experiments and bets

CABIN is actively seeking profitable opportunities which are mission-aligned with their objectives.

Proposals for experimental projects are submitted for approval and they all have budget, deliverables and a timeframe.

Some of the previous bets:

Software for Network States - selling software
Deep Work Club - paid for retreats in the "neighbourhoods"
Supper Clubs - paid for community events

---

There's is so much more to what they do and how, especially in terms of Web3 technologies.

**Comment by asimong** [comment]
I find this an appealing vision, especially for the adolescent and young adult in us all. 

What I don't see is the stability and predictability that is needed, at least initially, to raise children.

Thus, I see this as a very good and useful phase. Do your CABIN stuff, live your CABIN life — preferably at a young age! If you don't want to settle and have a family, you can continue doing that. But if your calling to contribute to the world includes procreation (which is by nature co-procreation), then …
Decide what you want to commit to: what is yours to do in life. Stick with that, with your co-parents, with whoever is part of your close intimate circle providing the stability as well as the variety for your children. When they in turn start their CABIN life, you'll be free again...

**Reply by martinprogrammer** [reply]
@asimong 
I agree, it's not a unified theory of everything, but I found a few concepts (that I'll be trickling in gradually) that I'm personally interested in and find them useful for our little endeavour:

"**But reality is much higher bandwidth...**".
It's so beautifully vague, enigmatic and poetic- a bit like Hegel's musing on the idea of absolute knowing - "...**God becomes aware of Himself**".

I am also interested in "deepening relationships" and "serendipitous in-person collaboration".

**Reply by asimong** [reply]
For sure, "reality is much higher bandwidth"! Though I would say that's pretty precise, not just poetic :-)

**Comment by martinprogrammer** [comment]
> A world where people can live in flexible ways,
travel to different places,
have home in those places,
co-live with people who have aligned vision,
maintain this flexibility and
be part of multiple overlapping networks

Does this mean - a sub-world where people who want to live alternatively - will be able to, within the over-reaching infrastructure and paradigm of the mainstream world?

or

A "new" world where "flexible living" will also apply to current "mainstream" living where the conventional folk will be just one of the groups?

Is this distinction relevant at all?
How does this compare to "our" vision?

**Comment by catherinet1** [comment]
Thanks for the summary of this interesting project, @martinprogrammer.

Reading it, I find myself trying to get more of a grasp of what this looks like concretely at the moment. What is the status of the project? How much does membership cost? What locations do they currently have significant presence in? What kinds of people are participating in or drawn to this at the moment?

Perhaps the deeper questions beneath this are something like: Is this vision viable? How are they going to get there? Do I trust their roadmap?

Some of these questions I imagine I could find answers to by looking more at their website (could you add a link?) -- not expecting you to answer all of these questions; just raising what the discussion thread brings up in me.

**Reply by rufuspollock** [reply]
I would second @catherinet1.

For me I'd be interested in:

- What is CabinDAO (briefly)
- Any history
- What are they doing right now
- etc

I remember looking at them a couple of months ago when someone mentioned them on the whatsapp chat.

**Comment by martinprogrammer** [comment]
> Perhaps the deeper questions beneath this are something like: Is this vision viable? How are they going to get there? Do I trust their roadmap?

@catherinet1 

Their vision is much less ambitious in scope than the one we're currently pondering.

I deliberately picked this mission statement because it's very simple and cogent and doesn't require any massive foundational undertaking to make it possible:

***A world where people can live in flexible ways,
travel to different places,
have home in those places,
co-live with people who have aligned vision,
maintain this flexibility and
be part of multiple overlapping networks***

All of this people can do right now - the project is simply providing an infrastructure and a mechanism and community for these people to join.

It's a bit like a nudist movement?  The demand is there, but not easy to practice outside a specific environment.

The economical model is either subscription for the infrastructure or the hubs of economical verticals where people would work in co-ops.

That probably needs further development and experimentation, but nothing controversial or outside of ideological constraints (if any).

I'll comment on other questions separately ;-)
